tests/integration/nerdctl/gha-run.sh

```diff
@@ -58,6 +58,23 @@ function install_dependencies() {
 	sudo systemctl restart containerd
 }
 
+function collect_artifacts() {
+	if [ -z "${start_time:-}" ]; then
+		warn "tests start time is not defined. Cannot gather journal information"
+		return
+	fi
+
+	local artifacts_dir="/tmp/artifacts"
```

Comment:
@GabyCT , should a unique directory name be generated here, just in case we'll have two or more of these scripts running in parallel someday?

Reply:
@danmihai1 yes that is a good suggestion, let me think about it how we can do that and meanwhile I will merge this PR and submitted another to improve this situation as we need to gathered some logs at the moment